pkg/migrations/sql/20250401074946_initial.sql

```diff
@@ -0,0 +1,62 @@
+-- +goose Up
```

Comment:
Just a thought ...
So this file is actually duplicates the model structure declaration
Am I right?
So from now, any change need to be done in both places?
I wonder if there is a way to have it declared only in one file

Reply:
> So this file actually duplicates the model structure declaration
yes.
> So from now, does any change need to be done in both places?
no. Just in the SQL files. Gorm is no longer used to migrate the db. it's just, that we already have a db in stage so I used the `IF EXISTS` statement not to break the migrations.

Reply:
Can you please elaborate on this `IF EXISTS` statement, where is it checked?

Reply:
normally we don't have to use this statement but, in stage and cnv, we're trying to migrate that is already migrated with GORM, therefore without this statement, the migration will fail. 
The goal for this initial migration is to sync goose with what we already have.  For the next migrations, we don't need use this statement.